mlir/lib/Dialect/XeGPU/IR/XeGPUOps.cpp

```diff
@@ -667,6 +707,13 @@ LogicalResult LoadGatherOp::verify() {
   auto maskTy = getMaskType();
   auto valueTy = getValueType();
 
+  if (tdescTy && !tdescTy.isScattered())
+    return emitOpError("Expects a scattered TensorDesc.\n");
```

Comment:
nit: I don't think `emitOpError` string needs new line at the end

Reply:
removed